awx/ui_next/src/screens/Template/shared/JobTemplateForm.test.jsx

```diff
@@ -35,6 +43,7 @@ describe('<JobTemplateForm />', () => {
         handleCancel={jest.fn()}
       />
     );
+    expect(LabelsAPI.read).toHaveBeenCalled();
```

Comment:
instead of checking for the expected side-effect on the api function, can you check from the perspective of a user that `Sushi` or `Major` shows up somewhere in the contents of the rendered component we're testing? 

I'm thinking something along the lines of this:

```jsx
const wrapper = mountWithContexts(
    <JobTemplateForm
       template={mockData}
       handleSubmit={jest.fn()}
       handleCancel={jest.fn()}
    />
  );

// might need to use a helper function so the promises resolve
expect(wrapper.find('MultiSelect Chip')[0].text()).toContain('Sushi'));
```

Reply:
👍